docs/learn/epoch/index.md

```diff
@@ -59,6 +59,18 @@ If not all orders can be executed, e.g. because there is not enough capital avai
 
 ## The Solver mechanism
 
```

Comment:
@annamehr, suggesting to add a sentence defining the Solver mechanism here. This helps introducing the chapter, and prevents the doubling of headings...